WIP: Image upload and use for software contributors, project image, team members and organisation logo

dmijatovic opened the pull request:
# Refactor image storage and reference

**This issue is work in progress.**

Closes #631 #635 

Changes proposed in this pull request:
*  Upgrade to Next v13
*  Images are stored centrally in one table. The id is SHA-1 of the image data. This addresses the caching in the most optimal way for an SPA (reloading of page is not required when user changes image).
 
How to test:
* `make start` to rebuild
* validate image uploads on all places:
* for software: contributors avatar upload, organisation logo upload
* for projects: project image, team member avatar upload, organisation logo upload
* for organisations: organisation logo upload, research unit organisation logo upload

PR Checklist:

*   [ ] Increase version numbers in `docker-compose.yml`
*   [ ] Link to a GitHub issue
*   [ ] Update documentation
*   [ ] Tests

cmeessen commented:
There is a caching and UX issue after removing the image of a contributor:

* create contributor
* add an image
* save
* edit contributor
* remove image
* there is no message that image was removed, save button is deactivated -> unclear whether image was actually removed?
* click cancel -> image is still shown
* refresh page -> now image is not displayed anymore

dmijatovic commented:
@cmeessen Thanks for spotting this! I have fixed this and rebased the solution. I also created new PR with updated solution. I am closing this PR in favor of #658
